README.md

```diff
@@ -122,42 +122,62 @@ The particle can be accesses using `iter->` (`*iter` is also possible).
 When created inside a OpenMP parallel region, work is automatically spread over all iterators.
 ```C++
 #pragma omp parallel
-for(auto iter = container.begin(); iter.isValid(); ++iter) {
+for(auto iter = autoPas.begin(); iter.isValid(); ++iter) {
   // user code:
   auto position = iter->getR();
 }
 ```
-### Simulation Workflow
-The simulation should always consist of the following phases:
-1. blub
-2. blob
-3. blab
-
-
-#### When it is necessary
-You have to update the container when the two conditions are fullfilled:
-* If you moved particles
-* You want to use `iteratePairwise()` or a RegionParticleIterator
-
-#### When it is not enough
-If you moved particles by more than one interaction length.
-If you are planning to move particles by a long distance,
-e.g. because of boundary conditions please delete the particles and add them again:
+### Simulation Loop
+One simulation loop should always consist of the following phases:
```

Comment:
would make sense to distinguish the MPI / no-MPI cases

Reply:
there really isn't much different there...